### ISSUE TITLE
Generating `rdf:type` triples on rows

### ISSUE THREAD

**Issue by JeniT** [w3c/csvw#459]
When generating RDF for a table, an `rdf:type` triple is generated with the value `csvw:Table`. Similarly for a table group. In standard mode, when nodes representing rows are created, should they be given a type `csvw:Row`?

**Comment by 6a6d74** [comment]
This would make sense - especially now that we have separated the "row" from the "subject" that the row describes - even when there's only a single subject; e.g.

```
@base <http://example.org/countries.csv> .
@prefix csvw: <http://www.w3.org/ns/csvw#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

_:d4f8e548-9601-4e41-aadb-09a8bce32625 a csvw:TableGroup ;
  csvw:table [ a csvw:Table ;
    csvw:url <http://example.org/countries.csv> ;
    csvw:row [ a csvw:Row ;
      csvw:rownum "1"^^xsd:integer ;
      csvw:url <#row=2> ;
      csvw:describes _:8228a149-8efe-448d-b15f-8abf92e7bd17
    ]
  ] .

_:8228a149-8efe-448d-b15f-8abf92e7bd17
  <#countryCode> "AD" ;
  <#latitude> "42.546245" ;
  <#longitude> "1.601554" ;
  <#name> "Andorra" .
```

I'll amend the csv2rdf doc tomorrow and prepare a PR to discuss at tomorrow's conf call.

**Comment by gkellogg** [comment]
+1

**Comment by iherman** [comment]
+1

Ivan

---

Ivan Herman
Tel:+31 641044153
http://www.ivan-herman.net

(Written on mobile, sorry for brevity and misspellings...)

**Comment by 6a6d74** [comment]
from the "+1" responses I'll assume that we want to enact this proposal. 

I've updated the csv2rdf spec and the examples used therein (`countries-standard.ttl`, `public-sector-roles-and-salaries-standard.ttl`, `events-listing-standard.ttl` and `tree-ops-ext-standard.ttl`); see PR #466. 

Outstanding actions include modification of the test data and any other places where "standard-mode" output is referenced from the specifications.

@gkellogg - please merge the PR when you're ready ...

**Comment by gkellogg** [comment]
Closed via PR #466
